src/pages/project/project.jsx

```diff
@@ -93,8 +97,8 @@ var Project = React.createClass({
         </div>
 
         <Menu fullWidth={this.state.params.mode === 'link'}>
-          { this.getRemovePageButton(isPlayOnly) }
-          <PrimaryButton url={ this.getPageURL(this.state.params, this.state.selectedEl) } off={isPlayOnly || !this.state.selectedEl} href="/pages/page" icon="../../img/pencil.svg" />
+          { this.getRemovePageButton(isPlayOrLink) }
```

Comment:
If we're renaming the var here, we need to rename it in [this function's argument list](https://github.com/errietta/webmaker-core/blob/errietta/link-new-page/src/pages/project/renderhelpers.js#L38), too, since we're no longer passing in "isPlayOnly".